src/auth/jwtclient.ts

```diff
@@ -26,13 +26,15 @@ import {
 } from './oauth2client';
 
 export interface JWTOptions extends RefreshOptions {
+  useJWTAccessAlways?: boolean;
```

Comment:
It would be nice to add descriptions to these properties so they show up in the documentation.  I know we don't have any existing ones here yet :/

Reply:
I'd like to advocate that we don;'t advertise this option in any interfaces, and just set it on the class.

This is what we did for `defaultScopes`.